charts/trino/values.yaml

```diff
@@ -40,6 +40,10 @@ imagePullSecrets: []
 server:
   workers: 2
   node:
+    # server.node.environment -- [node property configuration](https://trino.io/docs/current/installation/deployment.html#node-properties)
+    # The name of the environment. All Trino nodes in a cluster must have the same environment name. The name must
```

Comment:
Why do we have to inform users of this chart that all Trino nodes must have the same environment name, if the chart handles this, and chart users don't have any influence on it?